### ISSUE TITLE
Remove "experimental" designation from Alignment schema?

### ISSUE THREAD

**Issue by scharch** [airr-community/airr-standards#777]
Dunno if anyone is actually using it, but seems like we might want to have all of the schemas be "stable" for post-2.0 stasis

**Comment by javh** [comment]
Makes sense. Though, we'd also talked about maybe deprecating the whole schema, as it hasn't been kept up to date with Rearrangement.

**Comment by javh** [comment]
From the call:
* As Alignment is still experimental, we can just remove this for v2.0.
* Put in a PR to do so.
